helper/test_helpers.go

```diff
@@ -0,0 +1,33 @@
+package helper
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+type TestOpts struct {
+	app     abci.Application
+	chainId string
+}
+
+func NewTestOpts(app abci.Application, chainId string) *TestOpts {
```

Comment:
All these functions are only used in `broadcaster_test.go`. How about we move this file under `./broadcaster` and/or call it `broadcaster_util.go`?

Reply:
I tried that but it ends up causing an import cycle since this code will be imported in the `helper` package (which is itself being imported in `broadcaster`)